style.css

```diff
@@ -1,74 +1,33 @@
 #container {
   width: 375px;
   height: 5410px;
-  background-color: #f7f7f9;
 }
 
-#template1 {
-  width: 375px;
-  height: 718px;
-}
-
-.toolbar {
-  width: 375px;
-  height: 92px;
+.bg_image {
+  background-image: url(./Images/header/header.png);
+  position: absolute;
+  width: 462px;
+  height: 587.15px;
+  top: 104px;
```

Comment:
- [ OPTONAL] I recommend you reduce your background image width to something around 360px, it will fit your webpage in a mobile screen very well.